### PR TITLE
Decaffeinate everything

### PR DESCRIPTION
**Pull request by miiila**
Let's standardise our libraries and use pure JS everywhere. 

Negative side effect - code coverage is not easily substitutable for JS, so coveralls support is temporarily switched off. 

Promising tool https://www.npmjs.com/package/jscoverage throws some random errors and I don't consider coveralls so important to block it.

@dariagrudzien If you are interested, you can have a look, what coffee script was about and what we were talking about.

### CODE REVIEW

**Comment by miiila** [comment]
[Decaf Overlap checker](https://trello.com/c/9xnRGqte/602-decaf-overlap-checker)

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "5"
```
Comment (abtris) [inline .travis.yml]: If you remove this, pls update README.md too.

File: .npmignore [hunk 2]
```diff
@@ -1,9 +1,6 @@
-src
 scripts
 test
 cov.info
 README.md
-example.coffee
```
Comment (nadade) [inline .npmignore]: why not include example.js in .npmignore? tbh, not sure why it was ignored in the first place...

Comment (miiila) [reply .npmignore]: Good point, I am not sure either, but it's not necessary to have it there.

**Pull request merged**